scss/_global-settings.scss

```diff
@@ -7,11 +7,11 @@ $asset-path: 'https://assets.ubuntu.com/sites/ubuntu/latest/u/';
 /// transparent to use throughout the site
 $transparent:           transparent !default;
 /// the theme's core brand colour
-$brand-color:           #efefef !default;
+$brand-color:           #f5f5f5 !default;
 /// light brand colour
 $brand-color-light:     lighten($brand-color, 48%) !default; ///TODO make this a lighter brand color
```

Comment:
This TODO comment here - has this been done by you just now? Or what? What did the comment mean? Maybe you should remove it - or fix it?